src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -26,22 +27,22 @@ public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new DateOfBirth("13/2/1988"), new Address("Blk 30 Geylang Street 29, #06-40"),
-                getTagSet("friends"), new Gender("male")),
+                getTagSet("friends"), new Gender("male"), new Uid()),
             new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
                 new DateOfBirth("11/5/1968"), new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"),
-                getTagSet("colleagues", "friends"), new Gender("female")),
+                getTagSet("colleagues", "friends"), new Gender("male"), new Uid()),
             new Person(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@example.com"),
                 new DateOfBirth("4/12/2002"), new Address("Blk 11 Ang Mo Kio Street 74, #11-04"),
-                getTagSet("neighbours"), new Gender("female")),
+                getTagSet("neighbours"), new Gender("male"), new Uid()),
```

Comment:
Gender should be `female`

```suggestion
                getTagSet("neighbours"), new Gender("female"), new Uid()),
```